src/create-theme-listener.js

```diff
@@ -0,0 +1,27 @@
+const PropTypes = require('prop-types');
+const channel = require('./channel');
+
+function createThemeListener(CHANNEL = channel) {
```

Comment:
I would keep the channel lower case, because this mapping from lower to upper doesn't makes sense or look good. Knowing that its a constant doesn't give you any benefits when using in this way.